modules/ROOT/pages/release-notes/release-notes503.adoc

```diff
@@ -38,10 +38,10 @@ To start using TinyMCE for Swing as your new rich text editor, the first step is
 
 Contact link:{contactpage}[customer support] to discuss pricing options for a customized TinyMCE for Swing integration for your specific use case.
 
-For more information on *TinyMCE for Swing* refer to the full link:{baseurl}/integrations/swing/[documentation].
+For more information on *TinyMCE for Swing* refer to the full xref:integrations/swing.adoc[documentation].
 
 Refer to this https://github.com/tinymce/tinymce-swing-codesamples[GitHub repo] for examples on how to use TinyMCE for Swing.
 
 == Updates and enhancements
 
-The new TinyMCE 5.0.3 editor comes with significant changes to the previous versions. Improved solutions have been introduced for unnecessary and obsolete configuration options. A comprehensive list of the updates and fixed issues is available in the link:{baseurl}/changelog/#version503march192019[Changelog].
+The new TinyMCE 5.0.3 editor comes with significant changes to the previous versions. Improved solutions have been introduced for unnecessary and obsolete configuration options. A comprehensive list of the updates and fixed issues is available in the xref:changelog.adoc#version503march192019[Changelog].
```

Comment:
This xref doesn't resolve as the `version503march192019` id doesn't exist on the relevant title in `changelog.adoc` (well technically it's using the old `a` elements). e.g:
```
[[version503march192019]]
== 5.0.3 - 2019-03-19
```

Reply:
done